server/plugin_release.go

```diff
@@ -543,6 +551,23 @@ func getPluginAsset(ctx context.Context, githubClient *GithubClient, owner, repo
 	}
 }
 
+func markTagAsPreRelease(ctx context.Context, githubClient *GithubClient, owner, repo, tag string, preRelease bool) error {
```

Comment:
Mind adding a unit test for this code path as well? Thanks!

Reply:
sure, I'll do. I'll have to write unit test for this method or cover the path for the method from this is executed?

Reply:
@avinashdhinwa just unit testing this method will suffice for now. I think covering the path that calls this method would be difficult or may not even be possible.